pkg/features/kube_features.go

```diff
@@ -163,6 +163,12 @@ const (
 	//
 	// Enable pods to consume pre-allocated huge pages of varying page sizes
 	HugePages utilfeature.Feature = "HugePages"
+
+	// owner: @screeley44
+	// alpha: v1.9
+	//
+	// Enable Block volume support in containers.
+	BlockVolumeSupport utilfeature.Feature = "BlockVolumeSupport"
```

Comment:
this should be probably only `BlockVolume` - all features enable support of something